pkg/kyverno/apply/command.go

```diff
@@ -296,7 +299,6 @@ func applyPolicyOnResource(policy *v1.ClusterPolicy, resource *unstructured.Unst
 			fmt.Printf("\n%d. %s", i+1, r.Message)
 		}
 		fmt.Printf("\n\n")
-		responseError = true
```

Comment:
Is `responseError` indicating the operation response? Can we allow both? To print out the result and also return the error code?

Reply:
I was using "responseError" variable for checking:
- if any resource fails for mutation,
- or is a invalid resource 
- or is invalid for generation

In the above cases the CLI will exit with exit code 1. Also, CLI was not applying further policies on resources.

Now, I have made changes such that the CLI will apply all the provided policies on the provided resources.